source/reference/beta-program/abc-migration.txt

```diff
@@ -0,0 +1,80 @@
+.. _c2c-beta-abc-migration:
+
+==================
+A->B->C Migrations
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/beta-feature.rst
+
+.. include:: /includes/abc-migration-intro.rst
+
+Syntax 
+------
+
+To perform an A->B->C migration, run the following commands when starting 
+:ref:`mongosync <c2c-mongosync>`: 
+
+.. code-block:: shell
+
+   ./bin/mongosync \
+         --cluster0 <cluster-A-connection-string> \
+         --cluster1 <cluster-B-connection-string> \
+         --migrationName <string>
+
+   ./bin/mongosync \
+         --cluster0 <cluster-B-connection-string> \
+         --cluster1 <cluster-C-connection-string> \
+         --migrationName <string>
+
+To start the sync operation between the source and destination clusters, see 
+:ref:`c2c-quickstart-synchronize`.
+
+Behavior 
+--------
+
+If the first migration fails to finish committing before the second migration 
+starts, you might experience data loss.
```

Comment:
Addressing your comments: 

> Unpack "might" a little? Under which specific circumstances do you experience data loss?
Data loss can occur if the first migration operation doesn't commit before the second operation starts the commit process, which is the condition that's mentioned in the first half of the statement! This one's a bit dicey because with the Beta program, users are voluntarily accepting the [terms of use/disclaimer](https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-40077-abc-migration/reference/beta-program/#get-started), so during the download process and in our docs, they're made aware of the fact that they shouldn't use beta features on production environments **and** the fact that there's no stability guarantees. 

> Add a sentence or two on how to tell if you do experience data loss and how to remediate? This additional detail might be better in the limitations section.
I can potentially expand on this a bit more. @suspicious-mango do you have any suggestions on this? As for adding this in the limitations section, I discussed w/ Revathy and Johnny and they want to keep beta content contained within the Beta section and not within the GA docs to avoid any potentially conflicting information between GA and Beta setup.

Reply:
Thanks @ajhuh-mdb! This makes sense. Thanks for reminding about the beta terms of use. :) I I think it would be fine to add this as a warning, even just right here in the Behavior section.

Reply:
It also makes sense to keep beta docs in the beta docs section. Feel free to resolve as soon as you have follow up regarding how to remediate data loss.